llvm/test/CodeGen/AIE/aie2/GlobalISel/inst-select-vbcst.mir

```diff
@@ -23,7 +23,8 @@ body:             |
     ; CHECK-NEXT: $x0 = COPY [[VBCST_8_]]
     ; CHECK-NEXT: PseudoRET implicit $lr, implicit $x0
       %2:gprregbank(s32) = COPY $r0
-      %3:gprregbank(s32) = G_ASSERT_SEXT %2:gprregbank, 8
+      %4:gprregbank(s32) = G_CONSTANT i32 255
+      %3:gprregbank(s32) = G_AND %2, %4
```

Comment:
We should keep these tests and add two additional ones for the new patterns added by your PR